semantic-conventions/dev-requirements.txt

```diff
@@ -1,4 +1,4 @@
-black==21.8b0
+black==22.3.0
```

Comment:
A transitive dependency of `black` was failing the build, updating black to latest seems to have resolved it.